docs/source/configuration/authn-jwt.mdx

```diff
@@ -180,9 +181,13 @@ This is an array of possible token sources, as it could be provided in different
 </td>
 <td>
 
-Whether to ignore other prefixes in the `Authorization` header. If set to `false`, or unspecified, the router will only accept tokens with the prefix specified in `header_value_prefix`. If set to `true`, the router will ignore any requests that don't start with the prefix specified in `header_value_prefix`.
+By default, the Router errors when an unknown prefix is provided in the `Authorization` header. You must explicitly define prefixes in `header_value_prefix` or `sources`.
 
-If a header prefix is set to an empty string, this option is ignored.
+If you set `ignore_other_prefixes` to `true`, the Router allows unknown prefixes in the `Authorization` header and does not respond with an error when encountering one.
+
+If you set `ignore_other_prefixes` to `false` or leave it unspecified, the Router assumes the default behavior of responding with an error when it finds an unknown prefix in the `Authorization` header.
```

Comment:
I think starting with the default (and being able to refer to the default behavior just described) makes this a little easier to understand.

```suggestion
When `ignore_other_prefixes` is `false` (the default value), the router uses the default behavior and errors when it encounters an unknown prefix in the `Authorization` header.

If you set `ignore_other_prefixes` to `true`, the router allows requests with unknown prefixes in the `Authorization` header through and does not respond with an error when encountering one.
```